WhenPushComesToShove/Assets/3-Scripts/Modifiers/ActualModifiers/Knightmare.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Knightmare : BaseModifier
+{
+    Transform[] players;
+    
+    public override void Init()
+    {
+        players = GameState.players.ToArray();
+        Debug.Log(players.Length + " length");
+
+        foreach(Transform player in players)
+        {
+            player.GetChild(14).gameObject.SetActive(true);
```

Comment:
If we are going to actually have this in the game we shouldn't use get child 14 lol